.github/workflows/knip.yml

```diff
@@ -13,3 +13,4 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/prepare
       - run: pnpm lint:knip
+      - run: pnpm lint:knip:production
```

Comment:
I've been putting one command per workflow `.yml` when possible, so that we can get a granular display of which things are failing. But also these two commands are pretty related.

Personally I'd lean towards splitting these into two files, but I wouldn't block on it. What do you prefer?

Reply:
I don't mind either way. I'll split these workflows up.